core/src/main/java/com/novoda/noplayer/drm/DownloadDrmSessionCreator.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.noplayer.drm;
+
+import com.google.android.exoplayer2.drm.DrmSessionManager;
+import com.google.android.exoplayer2.drm.FrameworkMediaCrypto;
+
+import java.util.UUID;
+
+public class DownloadDrmSessionCreator implements DrmSessionCreator {
+
+    private static final UUID WIDEVINE_MODULAR_UUID = new UUID(0xEDEF8BA979D64ACEL, 0xA3C827DCD51D21EDL);
```

Comment:
these duplicated constants are all replaced with a single one inside  `DrmSessionCreator` in the next PR